### DISCUSSION TITLE
Interpretation of the wording of error messages und SC 3.3.1

### DISCUSSION THREAD

**Discussion by mraccess77** [w3c/wcag#1726] [Q&A]
SC 3.3.1 requires that you indicate the field in error and identify the error.

Say you have a field like the below that indicates required state:
* required
First Name: Input

After submitting without a name it now adds "Please enter your name" which was not there previously.  The text is associated with the form field programmatically.

* required
First Name: Input
Please enter your name

Does the wording "Please enter your name" clearly communicate there is an error present? or Does there need to be some other wording such as "incomplete", "missing",  or "error" to communicate to the user the field is in error?

If the text "please enter your name" were in red, would this make it more of an error under SC 1.4.1 as red makes it obvious to some there is an issue?

**Comment by mraccess77** [comment]
I have moved this to an issue as no one has responded in 3 weeks.

**Reply by patrickhlauke** [reply]
i was completely unaware that the (relatively new, to github) discussions feature was even being used...

**Reply by jake-abma** [reply]
me neither...!
